tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -2678,8 +2678,9 @@ else if(questionStatus == Status.FAILED)
 						UIOutput.make(tableRow, "forum-summary-widget-height", height);
 						//setting forums-messages url to get all recent messages for the site
 						UIOutput.make(tableRow, "forum-summary-site-url", myUrl() + "/direct/forums/messages/" + simplePageBean.getCurrentSiteId());
-						//setting this variable to redirect user to the particular message in the forums tool
-						UIOutput.make(tableRow, "forum-summary-view-url", myUrl() + "/portal/directtool/" + simplePageBean.getCurrentTool(simplePageBean.FORUMS_TOOL_ID));
+						//setting the url such that request goes to ShowItemProducer to display forum conversations inside Lessons tool
+						UIOutput.make(tableRow, "forum-summary-view-url", myUrl() + "/portal/hierarchytool/" + placement.getId()
```

Comment:
All the other URLs in this file use directtool links, is there some reason why that wouldn't work in this case?

`/portal/hierarchytool/` is an Oxford specific URL which won't exist in the main Sakai codebase, and doesn't fully exist in WebLearn 11.